packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -115,14 +116,37 @@ export default class OperatorModeStackItem extends Component<Signature> {
     }
   }
 
-  fetchIconURL = trackedFunction(this, async () => {
+  fetchRealmInfo = trackedFunction(this, async () => {
     let card = this.args.item.card;
     let realmInfo = await this.cardService.getRealmInfo(card);
-    return realmInfo?.iconURL;
+    return realmInfo;
   });
 
   get iconURL() {
-    return this.fetchIconURL.value ?? '/default-realm-icon.png';
+    return this.fetchRealmInfo.value?.iconURL ?? '/default-realm-icon.png';
+  }
+
+  get realmName() {
+    return this.fetchRealmInfo.value?.name;
+  }
+
+  @action
+  hoverOnRealmIcon() {
+    this.isHoverOnRealmIcon = !this.isHoverOnRealmIcon;
```

Comment:
Normally I’d be worried about this falling out of sync somehow since it’s just toggling vs being explicitly tied to enter vs leave but I couldn’t manage to break it 😯

Reply:
I'll keep an eye on this, if it breaks I'll update the implementation to make it tied with enter and leave.